en/customization/introduction.md

```diff
@@ -28,67 +28,81 @@ The aim of this service is to be able to offer all the dynamic contents of the a
+
 ```yml
 ["af", "am", "ar", "as", "az", "ba", "bg", "bn", "bo", "bs", "ca", "cs", "cy", "da", "de", "dv", "el", "en", "es", "et", "eu", "fa", "fi", "fil", "fj", "fo", "fr", "fr-CA", "ga", "gl", "gu", "ha", "he", "hi", "hr", "hsb", "ht", "hu", "hy", "id", "ig", "ikt", "is", "it", "iu", "iu-Latn", "ja", "ka", "kk", "km", "kmr", "kn", "ko", "ku", "ky", "ln", "lo", "lt", "lug", "lv", "lzh", "mg", "mi", "mk", "ml", "mn-Cyrl", "mn-Mong", "mr", "ms", "mt", "mww", "my", "nb", "ne", "nl", "nso", "nya", "or", "otq", "pa", "pl", "prs", "ps", "pt", "pt-PT", "ro", "ru", "run", "rw", "sk", "sl", "sm", "sn", "so", "sq", "sr-Cyrl", "sr-Latn", "st", "sv", "sw", "ta", "te", "th", "ti", "tk", "tlh-Latn", "tlh-Piqd", "tn", "to", "tr", "tt", "ty", "ug", "uk", "ur", "uz", "vi", "xh", "yo", "yua", "yue", "zh-Hans", "zh-Hant", "zu"]
 ```
+
 Of all the languages that Consul currently has defined (`available_locales`) in `config/application.rb` the only one that is not listed above and therefore no translation service is offered is Valencian `["val"]`.
 
-#### Pricing
+### Pricing
+
 The translation service used has the most competitive [pricing](https://azure.microsoft.com/en-us/pricing/details/cognitive-services/translator/).
 The price for each 1 Million characters translated is $10 and there is no fixed cost per month.
 
 Although technical measures have been taken to prevent misuse of this service, we recommend the creation of Alerts offered by Azure so that an Administrator can be notified in the event of detecting unusual use of the service. This service has a cost of $0.10 per month.
 
 To create an Alert in Azure we must follow the following steps:
+
 1. Sign in to the **Azure Portal**.
 1. Access the **Translator** service created earlier.
 1. Go to **Monitoring > Alerts** in the side menu:
-  1. Go to **Create alert rule**.
-  1. In **Select a signal** select `Text Characters Translated`.
-  1. Once selected we must define the logic of the Alert to suit our needs. Ex: Fill "Operator" field with "Greater than" option, fill "Aggregation type" field with "Total" option and fill "Threshold value" field with the number of characters we consider should be translated before being notified. In this section you can also set the time period and frequency of evaluation.
-  1. In order to be notified we have to create an **Action Group** and associate it with this Alert we're creating. To do this, access the button **Create** and fill out the form. As you can see there are different types of actions, we must select **Email/SMS/Push/Voice** and configure the option that we consider convenient according to our needs.
-  1. Once this group of actions has been created, it is directly associated with the rule we are creating.
-  1. Finally, all you have to do is add a name and click on the **Review + create**
-
-#### Add a new translation service
+
```

Comment:
I think that adding this line isn't necessary to follow the "Lists should be surrounded by blank lines" rule, and it makes it harder to read the list.

Reply:
I think this line was added in commit "MD022 Headers should be surrounded by blank lines".  

Maybe it would make sense to keep the line blank, right?

Reply:
The diff might be a bit confusing; I'm trying to comment on the blank line between these two:

```
1. Go to **Monitoring > Alerts** in the side menu:

   1. Go to **Create alert rule**.
```